public-api/db-cdr/generate-cdr/generate-survey-counts.sh

```diff
@@ -43,6 +43,9 @@ then
   exit 1
 fi
 
+COPE_PROJECT='aou-res-curation-prod'
+COPE_DATASET='SR2019q4r3_deid_io'
```

Comment:
I hardcoded these because when the cope survey is in prod data we do not need these variables since the usual project and dataset used for rest of the pipeline should be enough. Let me know what you think.